apps/monitoring/cveinfo/cveinfo.yaml

```diff
@@ -40,16 +40,19 @@ spec:
       VAULT_NAME: version-reporter-ptl
```

Comment:
Thanks @cpareek , sent you updates for the envs, kindly update Pr as well. Much appreciated